### PR TITLE
Do not report main functions args parameter in objects

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: detekt-cli/src/main/resources/default-detekt-config.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ test-pattern: # Configure exclusions for test sources
     - 'ForEachOnRange'
     - 'FunctionMaxLength'
     - 'TooGenericExceptionCaught'
-    - 'InstanceOfCheckForException'
```
Comment (vanniktech) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: Why was this removed?

Comment (arturbosch) [reply detekt-cli/src/main/resources/default-detekt-config.yml]: by accident, when I build detekt this line gets deleted, maybe need a clean build here, don't worry the last commit will get dropped :)
